manifests/0000_70_cluster-network-operator_03_deployment.yaml

```diff
@@ -31,6 +31,8 @@ spec:
           value: "quay.io/openshift/origin-sdn:4.3"
         - name: KUBE_PROXY_IMAGE
           value: "quay.io/openshift/origin-kube-proxy:4.3"
+        - name: KUBE_RBAC_PROXY_IMAGE
+          value: "quay.io/openshift/origin-kube-rbac-proxy:latest"
```

Comment:
Nit: Any specific reason this is latest and the other are pinned tags?

Reply:
No. I am not sure how it works ,  these get replaced by release manager I guess, typically like an art.yaml file that ties to image-references.yaml which will get replaced by an image name that the build system understands. Something like that.

Reply:
@juanluisvaladas I update the image tag  from latest to 4.3